src/parser/mage/fire/modules/features/Abilities.js

```diff
@@ -109,6 +109,15 @@ class Abilities extends CoreAbilities {
       },
 
       // Cooldowns
+      {
+        spell: SPELLS.TIME_WARP,
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        gcd: null,
+        cooldown: 300,
+        castEfficiency: {
+          disabled: true,
+        },
+      },
       {
```

Comment:
```JS        
castEfficiency: {
          disabled: true,
        },
``` 
is unnecessary code that you can just remove from all 3 abilities.js

Reply:
I think it is meant to do that because time warp is a spell that you do not care about the efficiency of at all, so having it say "can be improved" because you're not the designated lust caster or something like that would be incorrect